core/src/tx_sender/mod.rs

```diff
@@ -1,11 +1,40 @@
-use bitcoin::taproot::TaprootSpendInfo;
-use bitcoin::TapNodeHash;
-use bitcoin::XOnlyPublicKey;
-use eyre::OptionExt;
-
-use bitcoin::{Amount, FeeRate, OutPoint, Transaction, TxOut, Txid, Weight};
-use bitcoincore_rpc::{json::EstimateMode, RpcApi};
-use serde::{Deserialize, Serialize};
+//! # Transaction Sender
+//!
+//! Transaction sender is responsible for sending Bitcoin transactions, bumping
+//! fees and making sure that transactions are finalized until the deadline, all
+//! in background.
+//!
+//! ## Concepts
+//!
+//! ### Transaction/UTXO Activation and Cancelation
+//!
+//! Transactions and UTXOs can be marked as active, non-active or cancelled.
+//!
+//! Transactions/UTXOS that are marked as active are candidate transactions for
```

Comment:
what does it mean for a UTXO to be active?

Reply:
activation is only used for transactions, not UTXOs. UTXOs can be spent which trigger a condition.